frontend/src/pages/article/ArticlePage.tsx

```diff
@@ -9,14 +9,20 @@ import useArticleList from "./hooks/useArticleList";
 import TagList from "./TagList/TagList";
 
 function ArticlePage() {
-  const { isLoading, articles } = useArticleList();
-  const articleCategoryLabels = Object.values(ARTICLE_CATEGORY_MAP).map(
-    (item) => item.label,
-  );
+  const { refetch, isLoading, articles } = useArticleList();
+
+  const articleCategoryItems = ARTICLE_CATEGORY_ENTRY.map(([key, value]) => ({
+    key,
+    label: value.label,
+  }));
```

Comment:
```suggestion
  const articleCategories = ARTICLE_CATEGORY_ENTRY.map(([key,{ label }]) => ({ key, label }));
```
사소하지만 카테고리 표현할 때 이전에도 `categories`로 표현한 것 같아서 일관성 측면에서 통일해도 좋을 것 같고, 디스트럭처링 문법을 통해 가독성을 개선할 수 있을 것 같습니다!